service/app/controllers/organisationmanagement/OrganisationController.java

```diff
@@ -161,7 +166,9 @@
 
     try {
       JsonNode requestData = request().body().asJson();
-      ProjectLogger.log("Get Organisation details request: " + requestData, LoggerEnum.INFO.name());
+      ProjectLogger.log(
+          "OrganisationController:getOrgDetails call start with requested data =  " + requestData,
+          LoggerEnum.INFO.name());
```

Comment:
"OrganisationController: getOrgDetails called with data =  "
Change to DEBUG level